www/temporary-cdn.css

```diff
@@ -0,0 +1,332 @@
+/***************************
+//Font Type
+***************************/
+@font-face { font-family: "rigid-square"; src: url("https://use.typekit.net/af/6727e5/00000000000000007735c42c/30/l?primer=7fa3915bdafdf03041871920a205bef951d72bf64dd4c4460fb992e3ecc3a862&fvd=n6&v=3") format("woff2"), url("https://use.typekit.net/af/6727e5/00000000000000007735c42c/30/d?primer=7fa3915bdafdf03041871920a205bef951d72bf64dd4c4460fb992e3ecc3a862&fvd=n6&v=3") format("woff"), url("https://use.typekit.net/af/6727e5/00000000000000007735c42c/30/a?primer=7fa3915bdafdf03041871920a205bef951d72bf64dd4c4460fb992e3ecc3a862&fvd=n6&v=3") format("opentype"); font-display: auto; font-style: normal; font-weight: 600; font-stretch: normal; }
+
+@font-face { font-family: "rigid-square"; src: url("https://use.typekit.net/af/bb5c38/00000000000000007735c433/30/l?primer=7fa3915bdafdf03041871920a205bef951d72bf64dd4c4460fb992e3ecc3a862&fvd=n7&v=3") format("woff2"), url("https://use.typekit.net/af/bb5c38/00000000000000007735c433/30/d?primer=7fa3915bdafdf03041871920a205bef951d72bf64dd4c4460fb992e3ecc3a862&fvd=n7&v=3") format("woff"), url("https://use.typekit.net/af/bb5c38/00000000000000007735c433/30/a?primer=7fa3915bdafdf03041871920a205bef951d72bf64dd4c4460fb992e3ecc3a862&fvd=n7&v=3") format("opentype"); font-display: auto; font-style: normal; font-weight: 700; font-stretch: normal; }
+
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@300;400;500;600&display=swap&subset=latin');
```

Comment:
Just to formalize my last comment, this line needs to be moved to the top of this file (before the '@font-face's). Even if this file is going to be replaced, it'll make the changes to review locally (so others can take a look and sign off before this file is removed)

Reply:
I think the font size needs to be smaller to prevent wrapping. 

This is what it looks like after this change has been applied and the the Inter font loaded:

Firefox
![Uploading Screenshot 2024-08-22 at 12.26.59 PM.png…]()

Chrome
<img width="1202" alt="Screenshot 2024-08-22 at 12 26 48 PM" src="https://github.com/user-attachments/assets/38c9d076-12d9-45e6-aeba-70cc360b04e7">

Reply:
fixed